sb3_contrib/ppo_mask/ppo_mask.py

```diff
@@ -11,7 +11,7 @@
 from stable_baselines3.common.buffers import RolloutBuffer
 from stable_baselines3.common.callbacks import BaseCallback, CallbackList, ConvertCallback
 from stable_baselines3.common.on_policy_algorithm import OnPolicyAlgorithm
-from stable_baselines3.common.policies import BasePolicy
+from stable_baselines3.common.policies import BasePolicy, register_policy
```

Comment:
it seems you are using old version of SB3, `register_policy` was removed in 1.6.0